softdevice_controller/CHANGELOG.rst

```diff
@@ -27,12 +30,16 @@ Changes
 Bug fixes
 =========
 
+* Fixed an issue where a slave connection could disconnect prematurely if there were scheduling conflicts with other roles (DRGN-15469).
```

Comment:
Can we use peripheral instead of slave?

Reply:
The controller terminology as defined by the Bluetooth specification uses "master" and "slave". The next specification release will change this terminology. Until then I would prefer to use "slave" as done in the rest of this document.

Reply:
ok